spring-cloud-alibaba-examples/governance-example/label-routing-example/label-routing-default-provider-version-example/src/main/java/com/alibaba/cloud/examples/ProviderApplication.java

```diff
@@ -48,7 +44,10 @@ class Controller {
 
 		@GetMapping("/test")
 		public String test() {
-			return nacosRegistration.toString();
+			String host = nacosRegistration.getHost();
+			int port = nacosRegistration.getPort();
+			String version = nacosRegistration.getMetadata().get("version");
+			return "Route in " + host + ": " + port + ",version is " + version;
```

Comment:
the comment is `Route in 30.233.48.102: 18082,version is v1`, please modify to `Route in 30.233.48.102:18082, version is v1.`

Reply:
done